spec/system/user_show_integrations_spec.rb

```diff
@@ -0,0 +1,64 @@
+require 'rails_helper'
+
+RSpec.describe 'user show page tests', type: :system do
+  test_user = User.create!(name: 'Test User', bio: 'With a bio')
+  post_one = Post.create!(title: 'Test Post 1', text: 'With some text 1', author: test_user)
+  post_two = Post.create!(title: 'Test Post 2', text: 'With some text 2', author: test_user)
+  post_three = Post.create!(title: 'Test Post 3', text: 'With some text 3', author: test_user)
+  post_four = Post.create!(title: 'Test Post 4', text: 'With some text 4', author: test_user)
+  post_five = Post.create!(title: 'Test Post 5', text: 'With some text 5', author: test_user)
+
+  describe 'user show page' do
+    it 'check that show page loads the user name' do
+      visit user_path(test_user)
+      expect(page).to have_content(test_user.name)
+    end
+
+    it 'check that show page loads the user post count' do
+      visit user_path(test_user)
+      expect(page).to have_content('Number of posts:')
+      expect(page).to have_content(test_user.posts_count)
+    end
+
+    it 'check that user page shows the profile picture' do
+      visit user_path(test_user)
+      expect(page).to have_css('div.profile-picture')
+    end
+
+    it 'check that user page shows the user bio' do
+      visit user_path(test_user)
+      expect(page).to have_content(test_user.bio)
+    end
+
+    it 'check that user page shows the last 3 posts' do
+      visit user_path(test_user)
+      expect(page).to have_content(post_three.title)
+      expect(page).to have_content(post_four.title)
+      expect(page).to have_content(post_five.title)
+      expect(page).not_to have_content(post_one.title)
+      expect(page).not_to have_content(post_two.title)
+    end
```

Comment:
- [ ] Well done for creating a `spec` file which only tests the user show page. However, for this particular test,  you need to write tests that asserts that you can see the user's first 3 posts not their last 3 posts as you have done here 👍 
![image](https://github.com/wickathou/mcv-blog/assets/46839548/cd6cf799-2c93-4ecf-898f-1a42aca8f68c)